src/main/java/com/rapid7/client/dcerpc/mssamr/SecurityAccountManagerService.java

```diff
@@ -22,12 +22,12 @@
 import java.util.EnumSet;
 import com.hierynomus.msdtyp.AccessMask;
 import com.hierynomus.msdtyp.SID;
+import com.hierynomus.protocol.commons.EnumWithValue;
+import com.rapid7.client.dcerpc.RPCException;
 import com.rapid7.client.dcerpc.mssamr.messages.*;
-import com.rapid7.client.dcerpc.mssamr.objects.AliasHandle;
-import com.rapid7.client.dcerpc.mssamr.objects.DomainHandle;
-import com.rapid7.client.dcerpc.mssamr.objects.ServerHandle;
-import com.rapid7.client.dcerpc.mssamr.objects.UserHandle;
+import com.rapid7.client.dcerpc.mssamr.objects.*;
 import com.rapid7.client.dcerpc.objects.ContextHandle;
+import com.rapid7.client.dcerpc.objects.RPC_UNICODE_STRING;
```

Comment:
where is using RPC_UNICODE_STRING from this PR?

Reply:
It's not, thanks :)